contributors.json

```diff
@@ -159,5 +159,9 @@
   "WickedInvi": {
     "country": "United Kingdom",
     "name": "Deyan Petrov"
-  }
+  },
+  "RAYogeshwaran": {
```

Comment:
Let's order this in alphabetical order according to Github user name.

Reply:
ok, I will do.
Thanks for the support